GPUVerifyLib/GPUVerifyErrorReporter.cs

```diff
@@ -802,7 +802,7 @@ private static AccessType GetAccessType(CallCounterexample err)
             }
         }
 
-        private static QKeyValue GetAttributes(Absy a)
+        protected static QKeyValue GetAttributes(Absy a)
```

Comment:
Is it necessary to expose this? In the two locations where you use this, can't you just write `example.FailingCall.Attributes`?

Reply:
Reverting this change.